pkg/projects/state.go

```diff
@@ -182,6 +184,29 @@ func (state *state) GetDm() *metrics.DeviceMetric {
 	return state.deviceInfo.deviceMetric
 }
 
+//GetEVEIPs returns EVE IPs from info
+func (state *state) GetEVEIPs() (ips []string) {
+	if state.device.GetDevModel() == defaults.DefaultRPIModel {
```

Comment:
So why is this only for RPi? Why can't I use the same method when running under QEMU for example (note that with qemu we can actually use real networking -- not just 'user networking'). Of more concern is that I won't be able to use this with running EVE on GCP.

Reply:
I move logic into `eve.remote` option in config file.